AmethystAPI/src/amethyst/runtime/minecraft/MinecraftInfo.h

```diff
@@ -0,0 +1,19 @@
+#include "minecraft/src-deps/core/semVer/SemVersion.h"
+#include <string>
+#include <winrt/Windows.ApplicationModel.h>
+
+namespace Amethyst {
+    class MinecraftInfo {
+    public:
+        std::string name;
```

Comment:
Member variables should be prefixed with m. E.g mName for consistency with the rest of the project